### ISSUE TITLE
github actions nightly release broken since 1.3.0 

### ISSUE THREAD

**Issue by ryanmerolle** [netbox-community/netbox-docker#567]
## Current Behavior

Nightly release/check is not properly completing due to:

https://github.com/netbox-community/netbox-docker/runs/3492369296?check_suite_focus=true#step:3:788

## Expected Behavior

Quay & dockerhub updated with new tags if netbox core is updated.

## Debug Information

https://github.com/netbox-community/netbox-docker/runs/3492369296?check_suite_focus=true#step:3:788
https://github.com/netbox-community/netbox-docker/actions/runs/1193001221

**Comment by ryanmerolle** [comment]
```
#11 133.7   ERROR:root:Compiling the C Extension for the crc32c library failed. To enable building / installing a pure-Python-only version, set 'CRC32C_PURE_PYTHON=1' in the environment.
#11 133.7   error: command '/usr/bin/gcc' failed with exit code 1
```

**Comment by tobiasge** [comment]
The problem is this [issue](https://github.com/googleapis/python-crc32c/issues/83). Fix is merged into develop in PR #560.
So the builds will work again with the next minor release.

**Comment by ryanmerolle** [comment]
I thought the problem was this https://github.com/googleapis/python-crc32c/issues/92
